starknet_py/net/schemas/gateway.py

```diff
@@ -661,3 +650,18 @@ class SignatureOnStateDiffSchema(Schema):
     @post_load
     def make_dataclass(self, data, **kwargs) -> SignatureOnStateDiff:
         return SignatureOnStateDiff(**data)
+
+
+# Trace API schemas
+
+
+class BlockTransactionTracesSchema(Schema):
+    traces = fields.List(
+        fields.Nested(BlockSingleTransactionTraceSchema(unknown=EXCLUDE)),
```

Comment:
Do we need the unknown=EXCLUDE?

Reply:
That's the problem with gateway - we don't know (the code was already here, I just moved it to the bottom of the file 😅)